mdbenchmark/analyze.py

```diff
@@ -59,13 +59,12 @@
     show_default=True,
 )
 @click.option(
-    "-o",
-    "--output-name",
+    "-s",
+    "--save-csv",
     default=None,
-    help="Filename for the CSV file containing benchmark results.",
-    type=str,
```

Comment:
What happens if we call `mdbenchmark analyze --save-csv 1`? Will this be handles as an `int`, leading to a crash in the `if`-clause on line 96? Or does `click` handle this correctly, converting `1` to a `str`?

Reply:
click handles this correctly.